src/server/views/lists/partials/translators-interpreters/macros.njk

```diff
@@ -43,6 +43,11 @@
     {{ item.jsonData.servicesProvided.map(_.upperFirst).join(", ") if item.jsonData.servicesProvided }}
     {{ "Not provided" if not item.jsonData.servicesProvided or item.jsonData.servicesProvided.length === 0 }}
   </p>
+  {% if item.jsonData.swornTranslations or item.jsonData.swornInterpretations %}
+    <p class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1">
+      <strong>Official services</strong>: Sworn or certified translation, Sworn interpretation
```

Comment:
I believe this needs to be conditional so only show "Sworn or certified translation" If they have the sworn translation flag and only show "Sworn interpretation" if they have the sworn interpretation flag.

Reply:
Ah okay, I will double check this with Neil just to be sure 😉